lib/acme/client.rb

```diff
@@ -122,12 +126,14 @@ def kid
     @kid ||= account.kid
   end
 
-  def new_order(identifiers:, not_before: nil, not_after: nil)
+  def new_order(identifiers:, not_before: nil, not_after: nil, eab: nil)
     payload = {}
     payload['identifiers'] = prepare_order_identifiers(identifiers)
     payload['notBefore'] = not_before if not_before
     payload['notAfter'] = not_after if not_after
-
+    if eab
+      payload[:externalAccountBinding] = Acme::Client::Resources::Eab.new(eab, endpoint: endpoint_for(:new_order), jwk: @jwk).build_external_account_binding
```

Comment:
I see that you are adding `externalAccountBinding` to both newAccount and newOrder.

It's not in the spec. Any reason for doing that?